hole-punch-interop/impl/rust/v0.52/Makefile

```diff
@@ -1,5 +1,5 @@
 image_name := rust-v0.52
-commitSha := d99fa2f87406cbd1a207b29f8cbcec32b7b180f7
+commitSha := 1ead41ed04ce89e50b11a078586e47d89e17d3c4
```

Comment:
All the bugs around flakiness were in `rust-libp2p`. See the PR that this currently points to: https://github.com/libp2p/rust-libp2p/pull/4549